bettertest-kafka/src/main/java/org/bitintegro/bettertest/modules/kafka/testclient/KafkaMessageCollector.java

```diff
@@ -0,0 +1,7 @@
+package org.bitintegro.bettertest.modules.kafka.testclient;
+
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+
+public interface KafkaMessageCollector {
```

Comment:
I would suggest you to create package `internal` to mark classes that you use internally and don't expect to be used outside of the library.

Reply:
Please add JavaDoc.